index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="index.html">DAPHINE</a>
+        <div class="humburger-btn">
+          <span class="span-1"></span>
+          <span class="span-2"></span>
+          <span class="span-3"></span>
+        </div>
+      </nav>
+    </header>
+    <section class="headline__section">
+        <hgroup>
+          <h1><span class="Iam">I’m </span>Daphine</h1>
+          <h2>Glad to see you!</h2>
+        </hgroup>
+        <p>
+          I’m <span class="Iam2"> a software developer! </span>I can help you
+          build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded,
+          don’t hestiate to contact me.
+        </p>
+     <p class="connect">LET'S CONNECT</p>
+        <ul class="social">
```

Comment:
- [ ] Right now this section is displayed out in a `block` however it's one of the project requirements to layout this part using `flexbox` please fix it.

Reply:
Please be more specific, I am totally lost.

Reply:
Hello @Daphineatim I can see you have implemented the required change 🤗 I believe your question has been answered